lib/validation/validators/requiredYmd.ts

```diff
@@ -1,9 +1,6 @@
-import { YmdMap, type YmdType, isYmd } from '../../support/Day'
+import { type Ymd, YmdMap, type YmdType } from '../../support/Day'
+import { ymd } from './ymd'
 
 export function requiredYmd(value: unknown, required: YmdType[] = ['y', 'm', 'd']): boolean {
-  if (!isYmd(value, required)) {
-    return false
-  }
-
-  return required.every((r) => value[YmdMap[r]] != null)
+  return ymd(value, required) && required.every((r) => (value as Ymd)[YmdMap[r]] != null)
```

Comment:
Oh, wait, I found that it's written in the docs that "This validator will not validate if the values are valid numbers" ([ref](https://sefirot.globalbrains.com/validation/validators#requiredymd)).

Let me revert the change.

Reply:
Fixed: bb4e512